stm32h7/README.md

```diff
@@ -38,4 +38,4 @@ https://docs.rs/svd2rust/0.14.0/svd2rust/#peripheral-api
 
 | Module | Devices | Links |
 |:------:|:-------:|:-----:|
-| stm32h7x3 | STM32H743, STM32H753 | [RM0433](https://www.st.com/resource/en/reference_manual/dm00314099.pdf), [st.com](https://www.st.com/en/microcontrollers/stm32h743-microprocessors-753.html) |
+| stm32h7x3 | STM32H743, STM32H753 | [RM0433](https://www.st.com/resource/en/reference_manual/dm00314099.pdf), [st.com](https://www.st.com/en/microcontrollers-microprocessors/stm32h743-753.html) |
```

Comment:
This list is now incorrect; you need to update [stm32_part_table.yaml](https://github.com/stm32-rs/stm32-rs/blob/master/stm32_part_table.yaml#L328) to contain an entry for each generated module (`stm32h743`, `stm32h742v`, etc).

Reply:
Now resolved, I think

Reply:
Sorry, I meant that because we split these devices into multiple SVD files and therefore multiple modules and features in the stm32h7 crate, they each need their own higher-level entry in that YAML file, each with just one device member. This part of the README tells users which feature/module to use for a given device.

I think probably the list of features enabled for docs on stm32h7 in makecrates.py will need cutting down too -- having all four there will probably cause docs.rs to OOM kill the process. For other families we've picked just one or two devices that are representative of the family or are more widely used.

Reply:
Got it, there should be an entry under `sth32h7:` for each feature.

I've set it to only build stm32h743/stm32h743v for docs.rs (`743` seems more
common, on current ST dev boards etc.)